istioctl/cmd/istioctl/proxyconfig.go

```diff
@@ -208,6 +253,24 @@ func setupEnvoyConfigWriter(podName, podNamespace string, out io.Writer) (*confi
 	return cw, nil
 }
 
+func setupClustersEnvoyConfigWriter(podName, podNamespace string, out io.Writer) (*clusters.ConfigWriter, error) {
```

Comment:
Could you add the following todo comment above this function:
```
// TODO: migrate this to config dump when implemented in Envoy
// Issue to track -> https://github.com/envoyproxy/envoy/issues/3362
```

Reply:
have added, look forward to see eds in config_dump :)